ingest/Snakefile

```diff
@@ -101,45 +101,61 @@ rule clean:
 
 rule fetch_ncbi_dataset_package:
     output:
-        dataset_package="results/ncbi_dataset.zip",
+        dataset_package=directory("results/ncbi_dataset"),
     params:
         taxon_id=TAXON_ID,
         api_key=NCBI_API_KEY,
         gateway=f"--gateway-url {NCBI_GATEWAY_URL}" if NCBI_GATEWAY_URL else "",
         use_mirror=bool(MIRROR_BUCKET),
-        mirror_url=f"{MIRROR_BUCKET.strip('/')}/{TAXON_ID}.zip",
+        mirror_url=f"{MIRROR_BUCKET.strip('/')}/{TAXON_ID}.tar.zst",
+        unzip=unzip,
     shell:
         """
         if [[ "{params.use_mirror}" == "True" ]]; then
-            curl {params.mirror_url} -o {output.dataset_package}
+            curl {params.mirror_url} -o {output.dataset_package}.tar.zst
+            tar xvf {output.dataset_package}.tar.zst -C results
         else
             datasets download virus genome taxon {params.taxon_id} \
-            --no-progressbar \
-            --filename {output.dataset_package} \
-            --api-key {params.api_key} \
-            {params.gateway}
+                --no-progressbar \
+                --filename results/dataset.zip \
+                --api-key {params.api_key} \
+                {params.gateway} \
+            && {params.unzip} \
+                -o results/dataset.zip \
+                -d results
         fi
         """
 
 
 rule format_ncbi_dataset_report:
     input:
         script="scripts/format_ncbi_metadata.py",
-        dataset_package="results/ncbi_dataset.zip",
+        dataset_package="results/ncbi_dataset",
         config="results/config.yaml",
     output:
         ncbi_dataset_tsv="results/metadata_post_rename.tsv",
-    params:
-        unzip=unzip,
     shell:
         """
-        {params.unzip} -o {input.dataset_package} -d results
         python {input.script} \
             --config-file {input.config} \
-            --input results/ncbi_dataset/data/data_report.jsonl \
+            --input {input.dataset_package}/data/data_report.jsonl \
             --output {output.ncbi_dataset_tsv}
         """
 
+rule format_ncbi_dataset_sequences:
+    input:
+        dataset_package="results/ncbi_dataset",
+    output:
+        ncbi_dataset_sequences="results/sequences.fasta",
+    params:
+        unzip=unzip,
```

Comment:
Is this needed?

Reply:
Yeah we chop everything from the header line except for the id with ` -i, --only-id               print IDs instead of full headers`

Reply:
it was already there just moved up to a more logical place, see the deleted lines bottom left

Reply:
I meant this specific line with the unzip parameter - (but I don't really know snakemake)

Reply:
Ah yeah necessary ffor things to work on macOS, see above line 71-76


```
if os.uname().sysname == "Darwin":
    # Don't use conda-forge unzip on macOS
    # Due to https://github.com/conda-forge/unzip-feedstock/issues/16
    unzip = "/usr/bin/unzip"
else:
    unzip = "unzip"
```

Reply:
Thanks!

Reply:
guess we could name it `unzip_binary` to be more explicit

Reply:
aaah now I got it, you're right, this is no longer needed in this place!